Extensions/xkit_preferences.css

```diff
@@ -5,7 +5,24 @@
 	--xkit-on-primary: black;
 	--xkit-secondary: rgb(240,240,240);
 	--xkit-on-secondary: rgb(120,120,120);
-	--xkit-on-color: white;
+	--xkit-on-pale: black;
+	--xkit-on-saturated: white;
+
+	--xkit-05-overlay: rgba(0,0,0,.05);
+	--xkit-11-overlay: rgba(0,0,0,.11);
+	--xkit-22-overlay: rgba(0,0,0,.22);
+	--xkit-33-overlay: rgba(0,0,0,.33);
+	--xkit-44-overlay: rgba(0,0,0,.44);
```

Comment:
we're changing a lot of `-shadow` variable usage to `-overlay` in this PR. going forward, when would someone use `-shadow` vs using `-overlay`? is `-shadow` superfluous now?

Reply:
I changed some `--shadow` variables to `--overlay` for two reasons:

I replaced all `--shadow` variables that do not describe a box-shadow or text-shadow to be an `--overlay` instead. I did this for two reasons:

1. It's more descriptive this way. `--overlay` denotes something that will colour the entire element (ie hovering over a tab), whereas `--shadow` is now only used for shadows (ie text-shadows and box-shadows).
2. Dark Mode inverts the shadow colours (ie `--shadow` goes from black to white, and `--white-shadow` goes from white to black). But not all shadows should be inverted, such as `--shadow` (which goes from black to white) because it creates white box-shadows which look really odd because they're not as intuitive as black shadows). But some overlays are still useful, so I split them up into `--shadow` and `--overlay` so that I can invert `--overlay` while turning off `--shadow`.

If you don't do this (ie no `--overlay` variables and invert every shadow), then you get this result:

![xkit_dark_white_shadows](https://user-images.githubusercontent.com/42635772/103047032-52d9e500-45de-11eb-831c-970a401d01e1.png)

Weird white box-shadows.